docs/regmap/adi_regmap_dmac.txt

```diff
@@ -25,7 +25,7 @@ RO
 ENDFIELD
 
 FIELD
-[7:0] 0x61
+[7:0] 0x62
```

Comment:
Please update the field default value to the new standard, 0x00000062

Reply:
This field is only 8 bits wide, so 0x62 is correctly represented here. Are you referring to another field?

Reply:
The field width of 8 is correct and the representation of the value is also correct. However, in one of the meetings regarding the regmap structure, we decided to go with the 32 bit representation regardless of the field width. This update is not included yet in the documentation, the PR is still in review #1297 , under [docs/user_guide/ip_cores/creating_new_ip.rst](https://github.com/analogdevicesinc/hdl/blob/9e194fc2bd33c6d5bd9ba5b5eee4e41d1160e936/docs/user_guide/ip_cores/creating_new_ip.rst)

Reply:
Can we discuss here why this change is needed? In particular, why should every field value have a 32bit default value, even though it has a width of 8 bits as in this current example?

Reply:
We agreed to patch this in the #1297